### PR TITLE
HPCC-14702 Add logging service to HPCC repo

### PR DESCRIPTION
**Pull request by wangkx**
Signed-off-by: wangkx kevin.wang@lexisnexis.com

### CODE REVIEW

**Comment by wangkx** [comment]
@rpastrana please review.

**Comment by rpastrana** [comment]
looks good.

**Comment by wangkx** [comment]
@richardkchapman this PR is ready to be merged. Also please merge: https://github.com/hpcc-systems/LN/pull/762.

File: initfiles/componentfiles/configxml/wslogging.xsd [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+      <xs:sequence>
+        <xs:element name="MySQLLoggingAgents" maxOccurs="unbounded">
+          <xs:annotation>
+            <xs:appinfo>
+              <title>MySQL Logging Agents</title>
+            </xs:appinfo>
+          </xs:annotation>
+          <xs:complexType>
+            <xs:attribute name="MySQLLoggingAgent" type="mysqlloggingagentType" use="required">
```
Comment (richardkchapman) [inline initfiles/componentfiles/configxml/wslogging.xsd]: Is this relevant in the CE version?

**Comment by richardkchapman** [comment]
So the idea is that the logging service has moved to CE but the MySQL Logging agent is still EE? Is that right/sensible? Can the logging service be used without the MySQL logging component?

**Comment by richardkchapman** [comment]
@wangkx Some questions

File: initfiles/componentfiles/configxml/wslogging.xsd (hunk 1 above)
Comment (wangkx) [inline initfiles/componentfiles/configxml/wslogging.xsd]: Good catch. I should remove it from this file.

**Comment by wangkx** [comment]
For now, the logging service may not be used without the MySQL logging component. Some non-mysql based logging component may be added later to make the logging service work in CE.

**Comment by rpastrana** [comment]
The reason this was done was to get the logging service into CE in 6.0.0 for scapps migration. I think a licensing issues keeps us from providing the mysql implementation, but the plan in to provide a simple file based (or perhaps cassandra based) logging agent in CE: https://track.hpccsystems.com/browse/HPCC-14703

**Pull request merged**